docs/source/installation.rst

```diff
@@ -1,14 +1,17 @@
 Installation
 ============
 
-Installing Runway 
+Installing Runway
 ^^^^^^^^^^^^^^^^^
 
-Runway runs on Python 2.7 and Python 3.5+.
+Runway runs on Python 2.7 and Python 3.5+. It is also packaged as a
+single-binary bundled with python for distribution through non-python
+package managers.
```

Comment:
(not a blocker for this PR) prior to the 1.0 release, we should update this to be biased to curl/wget/iwr (more like "just download and run it") with more "advanced" deployments available via pip/npm.

Reply:
Unless we have the curl/wget/iwr endpoint, I'm inclined to leave it as-is and we can rewrite the block with the appropriate instructions when 1.0 is released

Reply:
Actually - @ITProKyle please do drop this doc change and we'll stage it in a separate branch/PR (need to time its merging to master AKA-docs-latest with binary publishing

Reply:
reverted changes to installation.rst